### DISCUSSION TITLE
Regular Updates and Security Patches for the Add-on

### DISCUSSION THREAD

**Discussion by niranjan94** [hassio-addons/addon-nginx-proxy-manager#460] [Q&A]
Hey Team,

Big thumbs up for your awesome work on this add-on! Thank you! 😄 

Just realized, it's been 7 months since the last release (November 2022). Meanwhile, the main nginx-proxy-manager has got a good number of bug fixes and better features.

This add-on is pretty key for a load of us who put it out there on the internet. So, having the latest security patches is pretty important.

I've seen that the required updates are already there in our repository. Just thinking, it'd be cool to have more regular tagging & releases.

Thanks for listening and keep up the good work!

Cheers!

**Comment by frenck** [comment]
The first update after this one, will be an hugely breaking change, I don't plan on continuing in the current state.

../Frenck

**Reply by niranjan94** [reply]
Hi Frenck! Thanks for the response :) Looking forward to the update

**Reply by HydrelioxGitHub** [reply]
I'm curious what will be released now I've seen there will be breaking changes .

**Comment by frenck** [comment]
A new release is pending, probably this weekend.

It uses a different, better maintainable, structure that allows for easier updates in the future.

../Frenck
